CWE/events/italy.txt

```diff
@@ -1,24 +1,59 @@
-#Italy becomes a republic
+#Abdication of King Vittorio Emmanuele III
 country_event = {
-  id = 8007704
-  title = EVT_8007704_NAME
-  desc = EVT_8007704_DESC
-  picture = "nwo2_italian_constitutional_referendum"
-  fire_only_once = yes
-
+	id = 8007757
+	title = "The Abdication of King Victor Emmanuel III"
+	desc= "EVTDESC7"
+	picture = "pavorotti"
+	
+	fire_only_once = yes
+	
 	trigger = {
 		tag = ITA
-		year = 1946 NOT = { year = 1950 }
 		government = hms_government
-		war = no
-		is_vassal = no
+		NOT = { year = 1950 }
+	}
+	
+	mean_time_to_happen = {
+		months = 6
+	}
+	
+	option = {
+		name = "Farewell Sciaboletta! Farewell!"
+		any_pop = { consciousness = 2 }
+		country_event = 8007704
 	}
 
-	mean_time_to_happen = { months = 6 }
+}
+
+#Italy becomes a republic
+country_event = {
+  id = 8007704
+  title = EVT_8007704_NAME
+  desc = EVT_8007704_DESC
+  picture = "nwo2_italian_constitutional_referendum"
+  is_triggered_only = yes
 
 	option = {
 		name = EVT_8007704_A
 		government = democracy
+		any_pop = { 
+				ideology = { factor = 0.20 value = conservative }
+			}
+		any_pop = { 
+				ideology = { factor = 0.15 value = big_tent }
+			}
+		any_pop = { 
+				ideology = { factor = 0.20 value = socialist }
+			}
+		any_pop = { 
+				ideology = { factor = 0.18 value = communist }
+			}
+		any_pop = { 
+			    limit = { OR = { ideology = communist } OR = { ideology = socialist } }
+				consciousness = 5
+				militancy = -5
+			}
+		set_country_flag = republic
```

Comment:
Please do not use country flags with just one generic word. Please use set_country_flag = ITA_1948_republic_flag instead